velox/functions/lib/window/NthValue.cpp

```diff
@@ -82,12 +87,24 @@ class NthValueFunction : public exec::WindowFunction {
     auto rawFrameStarts = frameStarts->as<vector_size_t>();
     auto rawFrameEnds = frameEnds->as<vector_size_t>();
 
+    vector_size_t nullsOffset = rawFrameStarts[0];
```

Comment:
this variable name is misleading because it may not point to the 'null' value.

Reply:
Changed.